spec/services/verifier_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'spec_helper'
+
+describe Verifier do
+  let(:parameters) do
+    {
+      :time => 1426064293.0,
+      :temp => 56.7,
+      :sensor_name => "0013a20040c17f5a",
+      :verification => 'c0ffee'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.